src/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsEventSourceLifetime.cs

```diff
@@ -21,6 +21,7 @@ public class TestsEventSourceLifetime
         /// </summary>
         [Fact]
         [PlatformSpecific(TestPlatforms.Windows)] // non-Windows EventSources don't have lifetime
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "ExerciseEventSource use private member reflection")]
```

Comment:
Is this a customer scenario? I am not sure if rd.xml helps here and don't know of a way to make test using private reflection to work. 
@morganbr